tests/integration/cmor/test_read_cmor_tables.py

```diff
@@ -1,6 +1,5 @@
 from pathlib import Path
 
-import pytest
```

Comment:
think you still need this one for `tmp_path`

Reply:
It's not needed, fixtures are automatically injected.

Reply:
To expand on that: if it were needed, the tests would be failing ;-)

Reply:
yeah I was wondering why it hasn't so far :laughing: